MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -116,24 +116,10 @@ open class BaseInstructionsBannerView: UIControl {
     }
     
     /**
-     Updates the instructions banner for a given `RouteProgress`.
+     Updates the instructions banner distance info for a given `RouteStepProgress`.
      */
-    public func update(for currentLegProgress: RouteLegProgress) {
-        let stepProgress = currentLegProgress.currentStepProgress
-        let distanceRemaining = stepProgress.distanceRemaining
-        
-        guard let visualInstructions = stepProgress.remainingVisualInstructions else { return }
-        
-        for visualInstruction in visualInstructions {
-            if stepProgress.distanceRemaining <= visualInstruction.distanceAlongStep || stepProgress.visualInstructionIndex == 0 {
-                
-                set(visualInstruction)
-                
-                stepProgress.visualInstructionIndex += 1
-                break
-            }
-        }
-        
+    public func updateDistance(for currentStepProgress: RouteStepProgress) {
```

Comment:
Add documentation

Reply:
@bsudekum that already has documentation 

https://github.com/mapbox/mapbox-navigation-ios/pull/1534/files/51a55587555605310ffac51f1f3c25e258c67cba#diff-7bae0cf60f54997d0d5e86fe02b5cddbR119